diagnostic.md

```diff
@@ -3,36 +3,40 @@
 What are three advantages of using Ember?
 
 ```sh
-# your answer here
+1) Fast
+2) Can have "back" button / bookmark, better ui overall.
+3) Data-binding
 ```
 
 What is the templating system used in Ember? How it is related to
 handlebars?
 
 ```sh
-# your answer here
+Handlebars...unless I am reading wrong...but you can add some more stuff. (I.e. components)
 ```
 
 Name at least two layers of an Ember application (1.0 or 2.0):
 
 ```sh
-# your answer here
+Model, Router, Template
 ```
 
 How does a component differ from an Ember template?
 
 ```sh
-# your answer here
+A component is reusable code. Template is specific to the route.
```

Comment:
Templates don't need routes in order to work. But yes, a component is reusable code, linked to a template, which can also be called from a different template.
.5/1